README.md

```diff
@@ -1,18 +1,20 @@
 # DC/OS Tools, Packages, and Installers for various platforms
 
-Tools for building DC/OS and launchign a cluster with it in the hardware of a customer's choice.
+Tools for building DC/OS and launching a cluster with it on a hardware of user's choice.
 
-  - *docker/* Locally defined docker containers packages are built in
+  - *docker/* Locally defined docker containers packages are built
+    here
   - *docs/* Documentation
-  - *ext/dcos-installer/* Backend for Web, SSH, and some bits of the Advanced installer. To be merged into the top codebase once the code is cleaned up
-  - *gen/* Python library for rendering yaml config files for various platforms into packages, with utilities to do things like make "late binding" config set by CloudFormation
+  - *ext/dcos-installer/* Backend for Web, SSH, and some bits of the
+    *Advanced Installer*.
```

Comment:
The note that this `ext/` folder is supposed to be temporary I believe is valuable